### PR TITLE
update packages

### PR DESCRIPTION
**Pull request by xakep139**

### CODE REVIEW

File: src/Orleans.Persistence.Cassandra/Orleans.Persistence.Cassandra.csproj [hunk 1]
```diff
@@ -3,8 +3,8 @@
     <TargetFramework>netstandard2.0</TargetFramework>
   </PropertyGroup>
   <ItemGroup>
-    <PackageReference Include="CassandraCSharpDriver" Version="3.6.0" />
-    <PackageReference Include="Microsoft.Orleans.Core" Version="$(OrleansPackageVersion)" />
-    <PackageReference Include="Microsoft.Orleans.Runtime.Abstractions" Version="$(OrleansPackageVersion)" />
+    <PackageReference Include="CassandraCSharpDriver" Version="3.9.0" />
+    <PackageReference Include="Microsoft.Orleans.Core" Version="2.3.0" />
+    <PackageReference Include="Microsoft.Orleans.Runtime.Abstractions" Version="2.3.0" />
```
Comment (ReubenBond) [inline src/Orleans.Persistence.Cassandra/Orleans.Persistence.Cassandra.csproj]: This value should be changed in [Directory.Build.props](https://github.com/OrleansContrib/Orleans.Persistence.Cassandra/blob/master/Directory.Build.props) instead

Comment (xakep139) [reply src/Orleans.Persistence.Cassandra/Orleans.Persistence.Cassandra.csproj]: Agreed

**Review by ReubenBond — APPROVED** [review]

**Comment by ReubenBond** [comment]
LGTM
@denisivan0v, are you able to take a look at this, too?

**Review by denisivan0v — APPROVED** [review]

**Comment by denisivan0v** [comment]
@xakep139 sorry for delay, @ReubenBond thanks for ping me.

**Pull request merged**